README.md

```diff
@@ -38,7 +38,9 @@ The recommended approach is deploying to Vercel. If you don't have an account al
 - Fork this repo
 - Create a new project on Vercel and connect this repo
 - It's a monorepo. So, choose either `packages/blog-starter-kit/themes/enterprise` or `packages/blog-starter-kit/themes/personal` as the root directory while importing on Vercel.
-  ![selecting the directory to deploy a monorepo](https://cdn.hashnode.com/res/hashnode/image/upload/v1695083263935/T5bByLxZT.png?w=500&h=800&auto=format)
+
+  ![selecting the directory to deploy a monorepo](https://github.com/victornwakpa/starter-kit/assets/61851642/f6d25a80-065a-48a2-9501-4047de544eb5)
```

Comment:
Hey @victornwakpa, could you upload this image to Hashnode's CDN using our uploader? https://hashnode.com/uploader

Reply:
okay

Reply:
I have applied the correction